src/pages/practice/100-prioritisation.md

```diff
@@ -0,0 +1,53 @@
+---
+templateKey: practice-page
+title: $100 Prioritisation
+subtitle: To help groups understand prioritise
+date: 2024-01-11T17:39:02.688Z
+authors:
+  - stellabella2022
+  - ""
+tags:
+  - methods
+  - insight
+mobiusTag: options
+icon: /images/100.webp
+whatIs: W﻿hen you need to narrow down your options or prioritise.  This is a
+  democratic relative prioritisation exercise where people get $100 each to
+  distribute across whatever needs prioritising.
+whyDo: P﻿rioritisaing is hard. This is a method that helps everyone be heard,
+  but also navigates you towards consensus in an equitable way.
+howTo: >-
+  ## M﻿ethod
+
+
+  1. F﻿ind a way of giving people $100.
+
+  2. P﻿eople can allocate their dollars however they want to. All on one item or spread their cash about!  They have to decide how much they value that thing and then use their dollars to represent that.  
+
+  3. The result is usually a well-spread priority list.
+
+
+  ## Tips:
+
+
+  * Recommend keeping allocations of money secret until people have finished working out where they are putting their $100 to avoid any tactical voting and maintain the suspense
+
+  * ﻿I often use monopoly money for this that I bought off eBay but my favourite is to create customer currency that personalises the exercise to the group e.g. Bella Bucks or Red Hat Rupeesed. You can also just use post-its! 
+
+  * W﻿hen sharing back the final results - I often convert to a % to help people understand final numbers more easily. 
+
+
+  ## V﻿ariations: 
+
+
+  1. You can force people to pick their clear top priority buy giving people certain  notes. For example 2x $50 means they can only pick two things, where 10x $10 gives them more flexibility to spread or back certain things more highly. My preference is combination of both with 1x$50, 1x$20, 2x$10 2x$5. 
+
+  2. D﻿epending on the group size there are variations where people share a $100 allowance. This can be the whole group, or pairs small group or pairs. This can make counting easier in large groups and also gives rise to some really good conversations. My preference is to use [1-2-4-All ](https://openpracticelibrary.com/practice/1-2-4-all/)practice because, like planning poker, the conversation is super valuable and drives alignment.
+mediaGallery:
+  - link: https://photos.app.goo.gl/3bv9XwEwiNgKAQ1Q7
```

Comment:
These links aren't showing up in the preview, so we probably need to add them to https://github.com/openpracticelibrary/opl-media or host them somewhere else. If you upload them here in a comment, I can add them, or maybe @tdbeattie can assist.

Reply:
I copied the pix and uploaded them to opl-media - I'll go fix one of the typos, and update the pic links now, but will still need your advice on whether the objective should be priorities, or prioritisation :)